frontend/src/app/recipe-list/filter.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from "@angular/core";
+@Pipe({
+  name: "filter",
+})
+export class FilterPipe implements PipeTransform {
+  transform(items: any[], searchText: string): any[] {
```

Comment:
Aren't these items recipes?
```suggestion
  transform(items: Recipe[], searchText: string): Recipe[] {
```